lib/msal-core/src/error/ClientAuthError.ts

```diff
@@ -102,9 +110,12 @@ export class ClientAuthError extends AuthError {
             `Cache error for scope ${scope}: ${ClientAuthErrorMessage.multipleMatchingAuthorities.desc}.`);
     }
 
-    static createPopupWindowError(): ClientAuthError {
-        return new ClientAuthError(ClientAuthErrorMessage.popUpWindowError.code,
-            ClientAuthErrorMessage.popUpWindowError.desc);
+    static createPopupWindowError(errDetail?: string): ClientAuthError {
+        var errorMessage = ClientAuthErrorMessage.endpointResolutionError.desc;
```

Comment:
this is a little duplicated could make small createErrMessage(description, errDetail? ) fn but its a nitpick not a big deal

Reply:
Resolving offline